packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -117,7 +118,16 @@ const Redirect = ({
             break;
         }
         case 'payment_withdraw': {
-            history.push(routes.cashier_withdrawal);
+            const is_wallet_account = loginid.toLowerCase().includes('w');
```

Comment:
Maybe we can pass `is_wallet_account` as a boolean from the useWalletsList instead of doing this ? 🤔

Reply:
@thisyahlen-deriv As we don't have access to hooks inside the `core` package, I used this solution to check if it's a wallet. 👍